src/pages/Home/container.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import Login from '../Login';
+import DashboardEmpty from '../DashboardEmpty';
+
+const Home = ({ userLoggedIn }) => (
+  userLoggedIn ? <DashboardEmpty /> : <Login />
+);
+
+Home.propTypes = {
+  userLoggedIn: PropTypes.string.isRequired
+};
+
+const mapStateToProps = (state) => (
+  { userLoggedIn: state.sessionReducer.session_id }
```

Comment:
Should I use reselect in this case?

Reply:
not now, just function

Reply:
Do I correctly understand your point?
1. Create selector
const isUserLoggedIn = (state) => state.sessionReducer.session_id;
2. Use this selector instead of mapStateToProps
export default connect(isUserLoggedIn)(Home);

Reply:
No, selector is an alternative to direct access to state, in current code it changes `state.sessionReducer.session_id` to `isUserLoggedIn(state)` to get the same value. You still need mapStateToProps to get values from redux.

Reply:
Thanks for the explanation.
Done